lua/cmp/entry.lua

```diff
@@ -333,8 +333,11 @@ end
 entry.get_replace_range = function(self)
   return self.cache:ensure({ 'get_replace_range', self.resolved_completion_item and 1 or 0 }, function()
     local replace_range
-    if misc.safe(self:get_completion_item().textEdit) and misc.safe(self:get_completion_item().textEdit.replace) then
+    local has_textEdit = misc.safe(self:get_completion_item().textEdit)
+    if has_textEdit and misc.safe(self:get_completion_item().textEdit.replace) then
       replace_range = self:get_completion_item().textEdit.replace
+    elseif has_textEdit and misc.safe(self:get_completion_item().textEdit.range) then
+      replace_range = self:get_completion_item().textEdit.range
     else
```

Comment:
Thank you for contribution. I think it's valid fix but I want to change it as the same as `get_insert_range`.

Reply:
Pushed a change.